qa/tasks/deepsea.py

```diff
@@ -1689,6 +1693,34 @@ def _repositories_to_remote(self, remote):
             args=args
             )
 
+    def _patch(self, **kwargs):
+        """Perform zypper patch with maintenance test repos from 'deepsea:patch' section
+
+        It will target all nodes in the cluster.
+
+        Repository format provided via deepsea-overrides.yaml in teuthology-suite.
+
+        deepsea:
+            patch:
+              - url: http://10.86.0.120/artifacts/ci/XYZ/SUSE_Updates_Storage_5_x86_64
+                name: SUSE_Maintenance_13609
+                prio: 1
+        """
+        if self.patch_repositories:
+            for repo in self.patch_repositories:
+                cmd = (
+                    "zypper -n addrepo --refresh --no-gpgcheck {url} {name} ; "
```

Comment:
Wouldn't it better to use `&&` instead of `;`, since you do not need to run all the next commands if previous fail?

Reply:
Sure